android/src/main/java/org/devio/rn/splashscreen/SplashScreen.java

```diff
@@ -85,6 +86,11 @@ public void run() {
                     videoView.getLayoutParams().height = fullHeight;
 
                     videoView.setVideoPath(videoPath);
+
+                    boolean allowAudio = options.hasKey("allowAudio") ? options.getBoolean("allowAudio") : false;
+                    if (!allowAudio) {
+                        videoView.setAudioFocusRequest(AudioManager.AUDIOFOCUS_NONE);
```

Comment:
note: i don't think this mutes the video, i think it just avoids muting other applications. Is that the intent?

Reply:
or do u want somethign like this? https://stackoverflow.com/questions/12435275/mute-a-playing-video-by-videoview-in-android-application

Reply:
I couldn't test properly because I don't have physical devices with me. The intent is muting the video, seems it should do it based on https://developer.android.com/reference/android/widget/VideoView#setAudioFocusRequest(int).

> Or use [AudioManager#AUDIOFOCUS_NONE](https://developer.android.com/reference/android/media/AudioManager#AUDIOFOCUS_NONE) to express that audio focus should not be requested when playback starts. You can for instance use this when playing a silent animation through this class, and you don't want to affect other audio applications playing in the background.

`setVolume` would work, but would it still take the audio focus off other apps? Our video is silent anyway, we want to make sure other audio is not paused when Exodus opens

Reply:
Just tested & my code works as expected on Android @mvayngrib

Reply:
You are right though that video is not muted, added that as well

Reply:
ecc5d7f57deee6b4ad85a77016958395941b20e7